pkg/features/kube_features.go

```diff
@@ -986,6 +994,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	KubeletTracing: {Default: true, PreRelease: featuregate.Beta},
 
+	LatePodHostnetHostPort: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
which do we want (impacts the polarity of the naming):
* start enabled and Beta, then graduate to GA, then go away
* start disabled and Deprecated, exist for "a while" until we get evidence it's safe to drop

if we stick with this name, does `Hostnet` read weirdly, rather than `HostNet`?

Reply:
Do we have a common pattern of gates meaning "EnableDeprecatedSomething" and trending from true->false as opposed to opposite?  I can't recall seeing one, but I don't object to it if we actually do it.

Reply:
Do you have a pref?  I could phrase it either way.  "Enable late defaulting" vs "Disable early defaulting"

Reply:
I'd lean toward something like `DefaultHostPortsInHostNetworkWorkloadObjects`, marked as deprecated, and defaulting to false. Relative terms like early/late seem harder to understand than explicitly saying what gets defaulted

Reply:
(but I only care about 2 out of 10 on the name / stage)

Reply:
flipped polarity in next push